src/components/atoms/UiTextarea/UiTextarea.vue

```diff
@@ -181,8 +192,10 @@ defineExpose(
     @include mixins.use-logical($element, padding, var(--space-12) var(--space-16));
     @include mixins.use-logical($element, border, 0);
 
+    margin: var(--space-2);
```

Comment:
Why 2px?

Reply:
if it's related to something maybe we should create an internal CSS var as the source of thought for both.
i.e. if it's based on border size and border-size will be changed then these values should be changed too

Reply:
Without these margins, it looks:
<img width="961" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/b1d578a9-973c-4182-adfa-a913b4dc5188">

Reply:
Do you talk about "resize" control?

Reply:
it's related to
```
.ui-textarea {
  $this: &;
  $element: textarea;

  @include mixins.inner-border($element, $radius: var(--border-radius-form));
...
```
our border covers this control. Maybe we should use "normal" border without this hack on `after` element? Maybe it solves all the issues?

Reply:
I checked, and without this `mixin` and adding a `border` to `textarea` again the drag handle is hidden and it doesn't look like in design:
<img width="784" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/7c361db7-ec6a-45be-bd52-100d9de2846a">